src/Node/FS/Async.purs

```diff
@@ -199,36 +198,19 @@ rmdir file cb = mkEffect $ \_ -> runFn2
 
 -- | Makes a new directory.
 mkdir :: FilePath
```

Comment:
If we're already going to break this, it may be better to provide an options record:

```purs
mkdir :: { dirname :: Filepath, recursive :: Boolean, perms :: Perms } -> Callback Unit -> Effect Unit
```

to avoid boolean blindness. What do you think?

Reply:
I'm not sure we need this. `mkdir` is pretty well-known from things like `bash` and those using JavaScript are already familiar with `mkdir { recursive: true }`. I'm not sure what else the boolean it could be.

Reply:
As discussed on the 0.15 call, we should just match the explicit API provided via Node:

https://nodejs.org/api/fs.html#fsmkdirpath-options-callback

```purs
mkdir :: FilePath -> { recursive :: Boolean, mode :: Perms } -> Callback Unit -> Effect Unit
```

Reply:
Done!

The code was already using `mkdir'`, so should we continue using that name to reduce breakage? Or name it the very verbose, `mkdirWithOptions`?

Reply:
I'm personally fine with the primed option.

Reply:
Ok. Can I get an approval then?